hugolib/node_as_page_test.go

```diff
@@ -385,6 +386,25 @@ title = "Deutsche Hugo"
 	assertFileContent(t, expectedFilePath(ugly, "public", "nn", "sect1", "regular2"), true,
 		"Single Title: Page 02")
 
+	// Page Params
+	pages := sites.findAllPagesByKind(KindPage)
+	for _, p := range pages {
+		if p.Path() == "sect1/regular1.en.md" {
```

Comment:
1. Good catch.  I need to use `filepath.Join`.

2. I didn't notice your standard.  Apparently, I have a different standard.  :grin: I use `assert` where I would use `t.Errorf` and `require` where I would use `t.Fatalf`.  I only use `Fatalf`/`require` to avoid accessing nil pointers and such in subsequent tests.  So, when I use `require`, it's a signal to me that we've got a serious problem.  If a Test func contains numerous assertions, I like to see them all fail instead of just the first one.  With `require` I'd only see the first failured assertion.  If you don't like my reasoning, I'll change to `require` in this file.

3. I agree, but I was trying to follow your lead.  :grinning:  Sometimes I'm not quite sure how to setup all the site scaffolding to run a simple unit test.  I'll take another look at the tests in this PR to see if I can pull them out into a more narrow test.